tests/cupyx_tests/scipy_tests/special_tests/test_expi.py

```diff
@@ -0,0 +1,41 @@
+import cupyx.scipy.special  # NOQA
+
+from cupy import testing
+from cupy.testing import numpy_cupy_allclose
+
+
+@testing.gpu
+@testing.with_requires("scipy")
+class TestExpi:
+
+    @numpy_cupy_allclose(scipy_name="scp")
+    def test_expi_zero(self, xp, scp):
+        return scp.special.expi(0.0)
+
+    @numpy_cupy_allclose(scipy_name="scp")
+    def test_expi_negative(self, xp, scp):
+        return scp.special.expi(-4.2)
+
+    @numpy_cupy_allclose(scipy_name="scp")
+    def test_expi_large_negative(self, xp, scp):
+        return scp.special.expi(-1000)
+
+    @numpy_cupy_allclose(scipy_name="scp")
+    def test_expi_positive(self, xp, scp):
+        return scp.special.expi(4.2)
+
+    @numpy_cupy_allclose(scipy_name="scp")
+    def test_expi_large_positive(self, xp, scp):
+        return scp.special.expi(1000)
+
+    @testing.for_dtypes(["f"])
+    @numpy_cupy_allclose(scipy_name="scp", rtol=1e-5)
+    def test_expi_linspace_float32(self, xp, scp, dtype):
```

Comment:
nit: I think you don't need to use the list argument in the `for_dtypes()` decorator...

Reply:
Yes, it's not really neccessary, added list just to supress dumb PyCharm warning.
![image](https://user-images.githubusercontent.com/29174280/187276110-d67ac0ab-592b-4750-bdf7-357eaf30b379.png)
Removed!
P.S. Maybe, we should add "Iterable" type annotation to `for_dtypes` function? There's an issue about type annotations #4831. Ofc, just for cuntributiors, not library users